Intersect.Client/Interface/Game/Crafting/CraftingWindow.cs

```diff
@@ -238,11 +238,13 @@ private void LoadCraftItems(Guid id)
                 var xPadding = mItems[i].Container.Margin.Left + mItems[i].Container.Margin.Right;
                 var yPadding = mItems[i].Container.Margin.Top + mItems[i].Container.Margin.Bottom;
 
-                var sizeFactor = (mItemContainer.Width - mItemContainer.GetVerticalScrollBar().Width) / (mItems[i].Container.Width + xPadding);
-                int widthFactor = sizeFactor * (mItems[i].Container.Width + xPadding) + xPadding;
-                int heightFactor = sizeFactor * (mItems[i].Container.Height + yPadding) + yPadding;
+                decimal sizeFactor = (mItemContainer.Width - mItemContainer.GetVerticalScrollBar().Width) /
+                                     (decimal)(mItems[i].Container.Width + xPadding);
```

Comment:
Why are you converting to a decimal?

Reply:
it was meant to calculate more precisely by using Math.Ceiling afterwards, but you are right, this is not really needed for this to work.

![imagen](https://user-images.githubusercontent.com/17498701/222324774-5c163e93-a433-4cf7-80a8-370286387fce.png)

```csharp
                var sizeFactor = (mItemContainer.Width - mItemContainer.GetVerticalScrollBar().Width) /
                                 (mItems[i].Container.Width + xPadding);

                mItems[i].Container.SetPosition(
                    i % sizeFactor * (mItems[i].Container.Width + xPadding) + xPadding,
                    i / sizeFactor * (mItems[i].Container.Height + yPadding) + yPadding
                );
```